api/build.gradle

```diff
@@ -25,13 +80,19 @@ dependencies {
   compile('org.springframework.boot:spring-boot-starter-web:+') {
     exclude module: 'spring-boot-starter-tomcat'
   }
+  compile 'org.springframework.security:spring-security-web:+'
```

Comment:
Below this can you please add:

      compile 'io.swagger:swagger-codegen:2.2.2'

It took me... too long to figure out that this was prevent VS Code from understanding the generated code.